src/tox_ansible/plugin.py

```diff
@@ -40,7 +40,6 @@
     "echo",
 ]
 ENV_LIST = """
-{integration, sanity, unit}-py3.7-2.9
 {integration, sanity, unit}-py3.8-{2.9, 2.12, 2.13}
```

Comment:
```suggestion
{integration, sanity, unit}-py3.8-{2.12, 2.13}
```
From here too? Or am I misunderstanding?

Reply:
I will keep ansible 2.9 for the other scenario for now, since this change was more related to removing of python 3.7 from the src and less to the ansible version.